extensions/BugModal/template/en/default/bug_modal/common_new_comment.html.tmpl

```diff
@@ -0,0 +1,55 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%#
+  # comment: comment object
+  # bug: bug object
+  #%]
+
+<div id="add-comment">
+
+  [% IF bug && !bug.check_can_change_field('longdesc', 0, 1) %]
+    <div id="new-comment-notice">
+      You are not allowed to make an additional comment on this [% terms.bug %].
+    </div>
+    [% RETURN %]
```

Comment:
it's more efficient to use [% ELSE %] than an early return.
RETURN is implemented in terms of raising an exception.